pmaweb/templates/team.html

```diff
@@ -2,196 +2,303 @@
+          change of how phpMyAdmin's internal class structure works as well as spearheading the
+          incorporation of the Twig template engine. He has quickly become familiar with the rest of the code base and has
+          offered many ideas for modernizing certain aspects of the code. He joined the team in 2017 and lives in Brazil.
+        </p>
+      </div>
+    </div>
+  </div>
 
-<div class="vcard" id="mauricio">
-<h3 class="fn">Maurício Meneghini Fauth</h3>
-<h4>Developer</h4>
-<img class="pull-left img-rounded img-margin" src="{% static 'images/heads/mauricio.jpg' %}" alt="Photo of Maurício Meneghini Fauth" />
-<p>
-Maurício's early work with the team began by being instrumental in designing and implementing a
-change of how <span class="org">phpMyAdmin's</span> internal class structure works as well as spearheading the
-incorporation of the Twig template engine. He has quickly become familiar with the rest of the code base and has
-offered many ideas for modernizing certain aspects of the code. He joined the team in 2017 and lives in Brazil.
-</p>
+  <div class="col-md-6 d-flex align-items-stretch">
+    <div class="card mb-3 w-100">
+      <div class="card-body">
+        <div class="d-flex align-items-center">
+          <a href="https://github.com/williamdes" target="_blank" rel="noopener noreferrer">
+            <img src="https://github.com/williamdes.png" class="card-img me-3" alt="William Desportes" style="width: 50px; height: 50px;">
+          </a>
+          <div>
+            <h5 class="card-title">William Desportes</h5>
+            <h6 class="card-subtitle mb-2 text-muted">Developer</h6>
+          </div>
+        </div>
+        <hr>
+        <p>
+          While doing application development, William used phpMyAdmin nearly every day and was attracted to the phpMyAdmin
+          project as a way to give back to the community and so that others can have a great tool to use as they learn the first steps of programming
+          with databases. One of his passions is to lower the barrier so new contributors can feel comfortable submitting a new pull request and he
+          enjoys coaching prospective contributors through seeing their work merged. William lives in France.
+        </p>
+      </div>
+    </div>
+  </div>
 </div>
 
-<div class="vcard" id="william">
-<h3 class="fn">William Desportes</h3>
-<h4>Developer</h4>
-<p>
-While doing application development, William used phpMyAdmin nearly every day and was attracted to the phpMyAdmin
-project as a way to give back to the community and so that others can have a great tool to use as they learn the first steps of programming
-with databases. One of his passions is to lower the barrier so new contributors can feel comfortable submitting a new pull request and he
-enjoys coaching prospective contributors through seeing their work merged. William lives in France.
-</p>
+<h2>Previous Team Members</h2>
```

Comment:
I've randomly noticed this section while looking and new website screenshots after being mentioned for the Weblate logo, and it seems unfair to me. Listing one randomly chosen previous team member and omitting people who spent years on the project is not the best approach. Better to remove previous members here and keep them only on https://docs.phpmyadmin.net/en/latest/credits.html.

Reply:
>Listing one randomly chosen previous team member and omitting people who spent years on the project is not the best approach.

I think this is mostly due to the fact that Dan never announced leaving the team. So that was me requesting to move him to a new section
But you are right, let's make a nice page for all of them that spent a lifetime maintaining phpMyAdmin
The link should be right after the team members in a visible way